service-worker.js

```diff
@@ -17,7 +21,7 @@ var preCachedRessources = [
 
 toolbox.precache(preCachedRessources);
 
-toolbox.router.get('/*', toolbox.cacheFirst);
-toolbox.router.get('/*', toolbox.cacheFirst, { origin: "cdnjs.cloudflare.com" });
-toolbox.router.get('/*', toolbox.cacheFirst, { origin: "cdn.jsdelivr.net" });
+toolbox.router.get('/*', toolbox.cacheFirst, contentCacheOptions);
```

Comment:
`networkFirst` or `fastest` - cacheFirst is never going to hit the network after first time.